index.md

```diff
@@ -58,44 +58,23 @@ Distinctions and Overlaps {#terms}
 
 Accessibility includes:
 
--   **Requirements that are technical and relate to the underlying code
-    rather than to the visual appearance** — for example, they ensure
-    that websites work well with assistive technologies such as screen
-    readers that read aloud web pages, screen magnifiers that enlarge
-    web pages, and voice recognition software that is used to input
-    text. These aspects are typically not a focus of usability research
-    and practice.
--   **Requirements that relate to user interaction and visual design** —
-    they are included in accessibility requirements because they can be
-    significant barriers to people with disabilities. For example,
-    understandable instructions and feedback for website forms and
-    applications is good usability; and is also an accessibility
-    requirement because many people with cognitive and learning
-    disabilities are otherwise excluded.
-
-Thus accessibility and usability overlap significantly. According to
-[ISO
-9241-11](http://www.iso.org/iso/catalogue_detail.htm?csnumber=16883),
-usability is defined as the "extent to which a product can be used by
-specified users to achieve specified goals effectively, efficiently and
-with satisfaction in a specified context of use". This could address
-accessibility ***when*** "specified users" includes people with a range
-of disabilities and "specified context of use" includes accessibility
-considerations such as assistive technologies. However, the needs of
-people with disabilities are often not sufficiently addressed in
-usability practice and research.
+-   **Requirements that are technical and relate to the underlying code rather than to the visual appearance**. For example, they ensure that websites work well with assistive technologies. This includes screen readers that read aloud content, and screen magnifiers that enlarge content. Voice recognitition software used to input text is another form of assistive technology.  These aspects are typically not a focus of usability research and practice.
+-   **Requirements that relate to user interaction and visual design**. Inadequate design can cause significant barriers for people with disabilities. That is why they are included. For example, understandable instructions and feedback for website forms and applications is good usability. They also help people with cognitive and learning disabilities. Without such requirements, some people with disabilities may be excluded from using the Web.
```

Comment:
Thanks @nitedog. I'm happy enough with these corrections.

Regarding impairment vs disability, we could probably have a long discussion on the merits of one over another :) I won't argue the toss, though in my background of disability rights and independent living, a lot of people view disability as something caused by the environment (e.g stairs), whereas the impairment is what the person has (paraplegia causing the use of a wheelchair). That's where I come from when I use impairment - not at all a medical perspective.

Regarding "cognitive or learning" vs "cognitive and learning", I won't argue about it either. But to explain the rationale, using the word "and" could lead some people to think both cognitive and learning are always tied together. Perhaps a better approach would be and/or but that becomes a mouthful.

Reply:
Thanks @vavroom.

Regarding disability vs impairment - I agree with you on the connotation of these terms. Since we want to emphasize the fact that *design* is what excludes people, especially people with cognitive and learning disabilities, I think the term "disability" is better in this paragraph and sentence.

Regarding and/or - I don't feel strongly but it threw me off because I'm so used to "people with cognitive and learning disabilities" as the common reference. Is this only me, or is there generally change towards using "people with cognitive or learning disabilities"?